web/src/components/AppBar/AppBar.vue

```diff
@@ -49,16 +49,24 @@
 
     <v-spacer />
 
-    <span v-if="!loggedIn" class="mr-1">
-      Want to create your own datasets?
-    </span>
-
     <template v-if="loggedIn">
+      <v-btn
+        :to="{ name: 'createDandiset' }"
+        exact
+        class="mx-3"
+        color="primary"
+        rounded
+      >
+        New Dandiset
+      </v-btn>
       <UserMenu
         v-if="loggedIn"
       />
```

Comment:
Since this `v-if` is redundant with the wrapping template:
```suggestion
      <UserMenu />
```